man/PPC-loo.Rd

```diff
@@ -175,26 +175,30 @@ compares computed PIT values to the standard uniform distribution. If
 calculated from the PIT values to the theoretical standard normal
 quantiles.}
 
-\item{eval_points}{For \code{ppc_loo_pit_ecdf()}, an optional integer defining
-the number of equally spaced evaluation points for the ECDF. Reducing
-\code{eval_poins} when using \code{interpolate_adj = FALSE} makes computing the
-confidence bands faster. If \code{pit} is supplied, defaults to \code{length(pit)},
-otherwise `min(nrow(yrep) + 1,}
+\item{K}{For \code{ppc_loo_pit_ecdf()} an optional integer defining the number
+of equally spaced evaluation points for the PIT-ECDF. Reducing K when
+using \code{interpolate_adj = FALSE} makes computing the confidence bands
+faster. If \code{pit} is supplied, defaults to \code{length(pit)}, otherwise
+\code{yrep} determines the maximum accuracy of the estimated PIT values and
+\code{K} is set to \code{min(nrow(yrep) + 1, 1000)}.}
 
-\item{prob}{For \code{ppc_loo_pit_ecdf()}, the desired simultaneous coverage
-level of the bands around the ECDF. A value in (0,1).}
+\item{prob, prob_outer}{Values between \code{0} and \code{1} indicating the desired
+probability mass to include in the inner and outer intervals. The defaults
+are \code{prob=0.5} and \code{prob_outer=0.9} for \code{ppc_loo_intervals()} and
+\code{prob = 0.99} for \code{ppc_loo_pit_ecdf()}.}
```

Comment:
I removed the use of the documentation template and just list the default of `ppc_loo_pit_ecdf()` separately.

Reply:
Looks good